bc/settings/third_party/threads.py

```diff
@@ -0,0 +1,9 @@
+import environ
+
+env = environ.FileAwareEnv()
+
+THREADS_APP_ID = env("THREADS_APP_ID", default="")
+THREADS_APP_SECRET = env("THREADS_APP_SECRET", default="")
+THREADS_CALLBACK_URL = env("THREADS_CALLBACK_URL", default="")
+THREADS_BUCKET_NAME = env("THREADS_BUCKET_NAME", default="")
+THREADS_BUCKET_ZONE = env("THREADS_BUCKET_ZONE", default="")
```

Comment:
Based on the default value in `put_object_in_bucket`, let's rename this setting to `THREADS_BUCKET_REGION`. (We should also update the .env.example file to reflect this change)

Reply:
[Done](https://github.com/freelawproject/bigcases2/pull/618/commits/f171af43be509313cec0fd7e6eb5a26dc6b9ec89)!